README.md

```diff
@@ -24,7 +24,7 @@ Options:
 
 1. This script requires unaltered recordings that have been directly downloaded from Pupil Cloud or exported from the phone. It will not work on recordings that have been opened in Pupil Player.
 2. Does not handle incomplete recordings (missing/corrupted files)
-3. Fixations detected locally might vary from those exported from Pupil Cloud. This is due to differences in real-time gaze inference when compared to the full 200 Hz gaze data available in the Cloud.
+3. Fixations and saccades detected locally might vary from those exported from Pupil Cloud. This is due to differences in real-time gaze inference when compared to the full 200 Hz gaze data available in the Cloud.
```

Comment:
We should add this change to the the changeset section below with an entry for next release, which is `1.0.11`